server/tests/routes/api/explore_test.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import DEFAULT
+from unittest.mock import patch
+
+from server.lib.nl.common.bad_words import EMPTY_BANNED_WORDS
+from server.lib.nl.explore.related import followup_with_geminipro
+from server.lib.nl.explore.utils import _RETRIES
+from web_app import app
+
+
+class TestExplore(unittest.TestCase):
```

Comment:
for test classes, should name it after the route you're actually testing so TestFollowupQuestions